content/docs/atom-guide/dev-guide/shaders/azsl/_index.md

```diff
@@ -445,24 +445,24 @@ For StructuredBuffer&lt;MyStruct&gt; case you'll get:
   
     float m_data;                     ; Offset:    0
     float4 m_arr[2];                  ; Offset:    4
-  
+```
 The [[pad_to(N)]] attribute is useful to pad the data with dummy variables until guaranteeing that the offset of the next variable starts with the desired alignment. The **N** argument is a single integral literal that must be a multiple of 4.  
 In the case posted above, `struct MyStruct`, using `[[pad_to(16)]]` can be useful to guarantee the same layout regardless of how it is used, as part of a `ConstantBuffer` or a `StructuredBuffer`:  
-  
+```cpp
```

Comment:
You can combine the last two code blocks into 1. 


Make the following edits to lines 454 to 456:
```
         float m_data;
         [[pad_to(16)]]       // This guarantees that the variable starts at offset 16.
         float4 m_arr[2];   
         [[pad_to(64)]]      // This guarantees that sizeof(MyStruct) is 64.
```

Reply:
I think it's important to keep them separate, even though they look similar.